tensorflow/c/kernels_experimental.h

```diff
@@ -59,6 +59,12 @@ TF_CAPI_EXPORT extern void TF_AssignVariable(
                      TF_Tensor* dest),
     TF_Status* status);
 
+TF_CAPI_EXPORT extern void TF_AssignRefVariable(
```

Comment:
Please add a `TF_Status* status` output argument to `TF_AssignRefVariable`, in line with other API entries.

Reply:
Done!